repo2docker/buildpacks/conda/__init__.py

```diff
@@ -356,9 +358,9 @@ def get_env_scripts(self):
                 (
                     "${NB_USER}",
                     r"""
-                mamba install -p {0} r-base{1} r-irkernel=1.2 r-devtools -y && \
-                mamba clean --all -f -y && \
-                mamba list -p {0}
+                ${{MAMBA_EXE}} install -p {0} r-base{1} r-irkernel=1.2 r-devtools -y && \
+                ${{MAMBA_EXE}} clean --all -f -y && \
```

Comment:
Looks like `micromamba clean` doesn't accept `-f`:

```
The following argument was not expected: -f
```

Reply:
Yes, we need to add this flag to micromamba clean but it should be pretty straightforward (`rm -rf $ROOT_PREFIX/pkgs` IIUC).

Reply:
You've been waiting ages on this PR, so I'm okay with removing `-f` here so we can merge the PR, and don't have to wait for a micromamba release.

Reply:
As you want. Getting this in will be awesome -- but we're happy to implement the missing `-f` flag, too: https://github.com/mamba-org/mamba/issues/1398

Reply:
Up to you! When I see that green checkmark, I'll merge, whether it's by removing `-f` or by updating µmamba.

Reply:
> You've been waiting ages on this PR, so I'm okay with removing -f here so we can merge the PR, and don't have to wait for a micromamba release.

Thanks ! This actually motivated some nice improvements in micromamba.